eng/Versions.props

```diff
@@ -147,6 +151,12 @@
     <SystemIdentityModelTokensJwtPackageVersion>5.3.0</SystemIdentityModelTokensJwtPackageVersion>
     <WindowsAzureStoragePackageVersion>8.1.4</WindowsAzureStoragePackageVersion>
 
+    <!-- Dependencies for Blazor. -->
+    <MicrosoftAspNetCoreBlazorMonoPackageVersion>0.8.0-preview1-20181126.1</MicrosoftAspNetCoreBlazorMonoPackageVersion>
```

Comment:
FYI - I've put these variables in 'pinned' because I don't think these are available in dependency flow automation yet.  That means you are on the hook to update this yourself, or wire up the repo which produces it into the automation.

Reply:
That's fine. Thanks for the info.